.circleci/config.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       - run:
           name: Lint bakery Python scripts
           command: |
-            flake8 bakery/src/**/**.py
+            flake8 bakery/src/**/**.py --max-line-length=100
```

Comment:
It looks like this is mostly to avoid breaking the regex string [here](https://github.com/openstax/output-producer-service/blob/e031b35c0ece56b204f9fd7bf3688a3f7b9f8213/bakery/src/scripts/link_single.py#L27) which makes sense. Maybe instead of a global change to line-length we can just add an in-line exception by appending that line with `# noqa: E501`? Or do we want to relax this linter rule more broadly anyways?

Reply:
Heh, yeah. I think that I've finally had it with the incredibly tiny max line length prescribed by E501. It would probably help if flake8 actually helped us format, but autopep8 barely helped in this case. What do you think?

Reply:
Sure, I have no problem if we want to increase the limit in general for the codebase, just wanted to check.